FBSnapshotTestCase/FBSnapshotTestController.m

```diff
@@ -126,25 +129,24 @@ - (BOOL)compareReferenceImage:(UIImage *)referenceImage
                     tolerance:(CGFloat)tolerance
                         error:(NSError **)errorPtr
 {
-  if (CGSizeEqualToSize(referenceImage.size, image.size)) {
-    BOOL imagesEqual = [referenceImage fb_compareWithImage:image tolerance:tolerance];
-    if (NULL != errorPtr) {
-      *errorPtr = [NSError errorWithDomain:FBSnapshotTestControllerErrorDomain
-                                      code:FBSnapshotTestControllerErrorCodeImagesDifferent
-                                  userInfo:@{
-                                             NSLocalizedDescriptionKey: @"Images different",
-                                             }];
-    }
-    return imagesEqual;
+  BOOL sameImageDimensions = CGSizeEqualToSize(referenceImage.size, image.size);
+  if (sameImageDimensions && [referenceImage fb_compareWithImage:image tolerance:tolerance]) {
+    return YES;
   }
```

Comment:
Yeah I saw #109 back when I made this PR and debated going ahead and adding that functionality. I was gonna err on the side of doing it in a second PR after this one to keep things focused.